pkg/operation/botanist/component/seedadmissioncontroller/seedadmissioncontroller.go

```diff
@@ -466,22 +466,61 @@ func GetValidatingWebhookConfig(caBundle []byte, webhookClientService *corev1.Se
 					},
 					Operations: []admissionregistrationv1.OperationType{admissionregistrationv1.Create, admissionregistrationv1.Update},
 				},
+			},
+			FailurePolicy:     &failurePolicy,
+			NamespaceSelector: &metav1.LabelSelector{},
+			ClientConfig: admissionregistrationv1.WebhookClientConfig{
+				CABundle: caBundle,
+				Service: &admissionregistrationv1.ServiceReference{
+					Name:      webhookClientService.Name,
+					Namespace: webhookClientService.Namespace,
+					Path:      pointer.String(extensionresources.EtcdWebhookPath),
+				},
+			},
+			AdmissionReviewVersions: []string{admissionv1beta1.SchemeGroupVersion.Version, admissionv1.SchemeGroupVersion.Version},
+			MatchPolicy:             &matchPolicy,
+			SideEffects:             &sideEffect,
+			TimeoutSeconds:          pointer.Int32(10),
+		}},
+	}
+
+	webhookConfig.Webhooks = getWebhooks(caBundle, webhookClientService, webhookConfig.Webhooks)
```

Comment:
Can you do the `append` here?
```suggestion
	webhookConfig.Webhooks = append(webhookConfig.Webhooks, getWebhooks(caBundle, webhookClientService))
```